Enhancements/TwilightPrincessHD_Bloom/rules.txt

```diff
@@ -0,0 +1,372 @@
+#name = 100% (Native)
+#condition = $bloom_enabled
+#default = 1
+#$bloom_blur_strength = 1.0
+
+
+# Area Specific Bloom Tint (49865bd2e62efda1)
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 0% (Same white bloom color everywhere)
+condition = $bloom_enabled
+$bloom_tint_strength = 0.0
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 10%
+condition = $bloom_enabled
+$bloom_tint_strength = 0.1
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 20%
+condition = $bloom_enabled
+$bloom_tint_strength = 0.2
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 30%
+condition = $bloom_enabled
+$bloom_tint_strength = 0.3
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 40%
+condition = $bloom_enabled
+$bloom_tint_strength = 0.4
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 50%
+condition = $bloom_enabled
+$bloom_tint_strength = 0.5
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 60%
+condition = $bloom_enabled
+$bloom_tint_strength = 0.6
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 70%
+condition = $bloom_enabled
+$bloom_tint_strength = 0.7
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 80% (Recommended)
+condition = $bloom_enabled
+$bloom_tint_strength = 0.8
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 90%
+condition = $bloom_enabled
+$bloom_tint_strength = 0.9
+
+[Preset]
+category = Area Specific Bloom Tint
+name = 100% (Native)
+condition = $bloom_enabled
+default = 1
+$bloom_tint_strength = 1.0
+
+
+# Shadow World Color Dampening (49865bd2e62efda1)
+# the alpha of the bloom color tint in the shadow world is used to darken the image
+# this is separated, because it is not affected by bloom strength at all
+
+[Preset]
+category = Shadow World Color Dampening
+name = 0% (Colors in shadow world as bright as in light world)
+$shadow_world_dampening = 0.0
+
+[Preset]
+category = Shadow World Color Dampening
+name = 10%
+$shadow_world_dampening = 0.1
+
+[Preset]
+category = Shadow World Color Dampening
+name = 20% (Recommended)
+$shadow_world_dampening = 0.2
+
+[Preset]
+category = Shadow World Color Dampening
+name = 30%
+$shadow_world_dampening = 0.3
+
+[Preset]
+category = Shadow World Color Dampening
+name = 40%
+$shadow_world_dampening = 0.4
+
+[Preset]
+category = Shadow World Color Dampening
+name = 50%
+$shadow_world_dampening = 0.5
+
+[Preset]
+category = Shadow World Color Dampening
+name = 60%
+$shadow_world_dampening = 0.6
+
+[Preset]
+category = Shadow World Color Dampening
+name = 70%
+$shadow_world_dampening = 0.7
+
+[Preset]
+category = Shadow World Color Dampening
+name = 80%
+$shadow_world_dampening = 0.8
+
+[Preset]
+category = Shadow World Color Dampening
+name = 90%
+$shadow_world_dampening = 0.9
+
+[Preset]
+category = Shadow World Color Dampening
+name = 100% (Native)
+default = 1
+$shadow_world_dampening = 1.0
+
+
+# Shadow World Desaturation (5f422bf63e25be7f)
+
+[Preset]
+category = Shadow World Color Desaturation
```

Comment:
I thought about combining both presets, but they are kinda vastly different. I myself wasn't sure for some time if I want to change both or just one, and which. So I thought more control equals better.

This one effects the actual brightness of the image. It uses the alpha value of the tint color, but only in the shadow world this ever is below 1.0.

In the end I turned this one down, to make the shadow world brighter to see better and make it less depressing, but kept the desaturation almost native, cause that help to make the shadow world more destinct.

Reply:
I more so meant that I'd like to provide users with easy, high quality options rather then having them figure it out on their own. I think having multiple recommended options is a great idea. I'm also not that big of a fan of recommending options, since monitors vary wildly and what looks good on your screen might look shit on other people their screens.